.storybook/themes/news.scss

```diff
@@ -1207,6 +1207,31 @@
 					gap: var(--global-spacing-6),
 					grid-template-columns: 2fr 1fr,
 				),
+				small-promo: (
+					display: grid,
+					gap: var(--global-spacing-5),
+					grid-template-columns: 1fr,
+					components: (
+						heading: (
+							font-size: var(--body-font-size),
+							line-height: var(--body-line-height),
+						),
+					),
+				),
+				small-promo-above: (
+					grid-template-columns: 1fr,
+				),
+				small-promo-below: (
+					grid-template-columns: 1fr,
```

Comment:
This is redundant as it's already set on the `small-promo`

Reply:
I removed the declaration